barretenberg/cpp/src/barretenberg/vm2/tracegen/lib/instruction_spec.cpp

```diff
@@ -0,0 +1,77 @@
+    { ExecutionOpCode::POSEIDON2PERM, { .subtrace_selector = SubtraceSel::POSEIDON2PERM, .subtrace_operation_id = 0 } },
+    // Execution
+    { ExecutionOpCode::SET, { .subtrace_selector = SubtraceSel::EXECUTION, .subtrace_operation_id = 1 } },
+    { ExecutionOpCode::MOV, { .subtrace_selector = SubtraceSel::EXECUTION, .subtrace_operation_id = 2 } },
+    { ExecutionOpCode::JUMP, { .subtrace_selector = SubtraceSel::EXECUTION, .subtrace_operation_id = 3 } },
+    { ExecutionOpCode::JUMPI, { .subtrace_selector = SubtraceSel::EXECUTION, .subtrace_operation_id = 4 } },
+    { ExecutionOpCode::CALL, { .subtrace_selector = SubtraceSel::EXECUTION, .subtrace_operation_id = 5 } },
+    { ExecutionOpCode::STATICCALL, { .subtrace_selector = SubtraceSel::EXECUTION, .subtrace_operation_id = 6 } },
+    { ExecutionOpCode::INTERNALCALL, { .subtrace_selector = SubtraceSel::EXECUTION, .subtrace_operation_id = 7 } },
+    { ExecutionOpCode::INTERNALRETURN, { .subtrace_selector = SubtraceSel::EXECUTION, .subtrace_operation_id = 8 } },
+    { ExecutionOpCode::RETURN, { .subtrace_selector = SubtraceSel::EXECUTION, .subtrace_operation_id = 9 } },
+    { ExecutionOpCode::REVERT, { .subtrace_selector = SubtraceSel::EXECUTION, .subtrace_operation_id = 10 } },
+    { ExecutionOpCode::SUCCESSCOPY, { .subtrace_selector = SubtraceSel::EXECUTION, .subtrace_operation_id = 11 } },
+    { ExecutionOpCode::RETURNDATASIZE, { .subtrace_selector = SubtraceSel::EXECUTION, .subtrace_operation_id = 12 } },
+    // Others
+    { ExecutionOpCode::SLOAD, { .subtrace_selector = SubtraceSel::OTHERS, .subtrace_operation_id = 0 } },
+    { ExecutionOpCode::SSTORE, { .subtrace_selector = SubtraceSel::OTHERS, .subtrace_operation_id = 1 } },
+    { ExecutionOpCode::NOTEHASHEXISTS, { .subtrace_selector = SubtraceSel::OTHERS, .subtrace_operation_id = 2 } },
+    { ExecutionOpCode::EMITNOTEHASH, { .subtrace_selector = SubtraceSel::OTHERS, .subtrace_operation_id = 3 } },
+    { ExecutionOpCode::NULLIFIEREXISTS, { .subtrace_selector = SubtraceSel::OTHERS, .subtrace_operation_id = 4 } },
+    { ExecutionOpCode::EMITNULLIFIER, { .subtrace_selector = SubtraceSel::OTHERS, .subtrace_operation_id = 5 } },
+    { ExecutionOpCode::L1TOL2MSGEXISTS, { .subtrace_selector = SubtraceSel::OTHERS, .subtrace_operation_id = 6 } },
+    { ExecutionOpCode::GETCONTRACTINSTANCE, { .subtrace_selector = SubtraceSel::OTHERS, .subtrace_operation_id = 7 } },
+    { ExecutionOpCode::EMITUNENCRYPTEDLOG, { .subtrace_selector = SubtraceSel::OTHERS, .subtrace_operation_id = 8 } },
+    { ExecutionOpCode::SENDL2TOL1MSG, { .subtrace_selector = SubtraceSel::OTHERS, .subtrace_operation_id = 9 } },
+    { ExecutionOpCode::DEBUGLOG, { .subtrace_selector = SubtraceSel::OTHERS, .subtrace_operation_id = 0 } },
+    { ExecutionOpCode::SHA256COMPRESSION, { .subtrace_selector = SubtraceSel::OTHERS, .subtrace_operation_id = 1 } },
+    { ExecutionOpCode::KECCAKF1600, { .subtrace_selector = SubtraceSel::OTHERS, .subtrace_operation_id = 2 } }
```

Comment:
Are the `subtrace_operation_id`s repeated for OTHERs here?

Reply:
ahh yeah - i should just delete these anyways until we find a proper subtrace for these.  We just need to support as many opcodes as we currently have in `EXEC_INSTRUCTION_SPEC` for now.